static/templates/copy_invite_link.hbs

```diff
@@ -0,0 +1 @@
+<a class="btn pull-right copy_invite_link" data-toggle="tooltip" title="{{t "Copy link" }}" aria-label="{{t "Copy link" }}" id='copy_generated_invite_link' data-clipboard-target="#multiuse_invite_link">{{> copy_to_clipboard_svg }}</a>
```

Comment:
You can add `copy_button_base` class here (class content in `zulip.css`) which automatically handles things like changing colour for on-hover, etc.

Reply:
Made those changes thanks for helping out 😄 @sumanthvrao

Reply:
Did you test it out locally? I have a feeling you would need to tweak the positioning to adjust for the new class.

Reply:
Yes I need to adjust it! The icon is sometimes changing its position?

Reply:
Yeah tweaked the position, tested with multiple links, working fine now. Thanks for mentioning this. @sumanthvrao